ee/server/service/software_installers_test.go

```diff
@@ -219,6 +219,25 @@ func TestSoftwareInstallerPayloadFromSlug(t *testing.T) {
 	assert.NotEmpty(t, payload.UninstallScript)
 	assert.True(t, payload.FleetMaintained)
 
+	// when SHA256 is no_check
+	ds.GetMaintainedAppBySlugFunc = func(ctx context.Context, slug string, teamID *uint) (*fleet.MaintainedApp, error) {
```

Comment:
This is a bit fragile, as hydration reaches out to GitHub rather than having the manifest served locally. Can leave it for now though since the most likely cause of this breaking is us starting to include hashes in Chrome for macOS, which will be an obvious functionality change that we should refactor tests for anyway..